counterpartylib/lib/database.py

```diff
@@ -128,6 +130,9 @@ def get_connection(read_only=True, foreign_keys=True, integrity_check=True):
             raise exceptions.DatabaseError('Could not perform integrity check.')
         logger.info('Integrity check completed.')
 
+    cursor.execute('PRAGMA auto_vacuum = 1')
+    cursor.execute('PRAGMA journal_size_limit = 0')
```

Comment:
this seems very low, esp. when setting it not just for kickstart but for everything. can we 1) set it higher and/or 2) set it to a different value juts for kickstart?

Reply:
hum.. the default is -1 (no limit) and 0 means the "always truncate rollback journals and WAL files to their minimum size".. not sure what the best.. I don't see how to decide on an arbitrary value otherwise..

Reply:
okay, it seems we only need this when using `PERSIST` mode, which we're just doing for `kickstart`, I think.

I think in that case we should set the `journal_size_limit` to `0` only for `kickstart`